tests/bgp/test_bgp_speaker.py

```diff
@@ -202,3 +204,179 @@ def test_bgp_speaker_announce_routes(common_setup_teardown, testbed, duthost, pt
                         "testbed_mtu": mtu },
                 log_file="/tmp/bgp_speaker_test.FibTest.log",
                 socket_recv_size=16384)
+
+
+@pytest.fixture(scope="module")
+def common_setup_teardown_v6(duthost, ptfhost, localhost):
```

Comment:
not sure why we have separate common_setup_teardown than ipv4.

i thought the bgp session are the same for both ipv4 and ipv6. only the route announce is different.

the concern is that there are many duplicate code between the common_setup_teardwon and this _v6 function.

Reply:
is this getting addressed?

Reply:
Yes. Please check the latest changes.